src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -24,6 +24,7 @@ class FlattenException
 {
     private $message;
     private $code;
+    private $title;
```

Comment:
I agree and I'm not even sure that this is a good idea. A `FlattenException` is not something you want to "configure", this is just a representation of an existing exception.